Include/Acidanthera/Library/OcPeCoffExtLib.h

```diff
@@ -27,6 +28,26 @@ PeCoffVerifyAppleSignature (
   IN OUT UINT32  *ImageSize
   );
 
+#ifdef EFIUSER
+
+/**
+  Verify Apple COFF legacy signature with pre-initialized image context.
+  Image buffer referenced via context is sanitized where necessary (zeroed),
+  and an updated length is returned through size parameter.
+
+  @param[in,out]  ImageContext    Image context.
+  @param[in,out]  ImageSize       Size of the image.
+
+  @retval EFI_SUCCESS on success.
+**/
+EFI_STATUS
+InternalPeCoffVerifyAppleSignatureFromContext (
+  IN OUT PE_COFF_LOADER_IMAGE_CONTEXT  *ImageContext,
+  IN OUT  UINT32                       *ImageSize
+  );
+
+#endif
```

Comment:
Here an just below: possibly slightly janky way of making these internals visible to user space utils only, or possibly okay?

Reply:
I think it’s ok, otherwise there only other option is a User-specific header. Declaring the function in the consumer code should not be done, as not all compilers will notice de-sync between prototypes without a shared header.

Reply:
> as not all compilers will notice de-sync between prototypes without a shared header

Yes